cypress/integration/api-class_spec.js

```diff
@@ -340,6 +340,8 @@ describe("Api class", () => {
 		});
 
 		it("should default referer to window.location.href", () => {
```

Comment:
Zou deze if niet binnen de `describe("subscribeDevice()", ...` moeten vallen. Nu lijkt deze test a.d.v. de beschrijving identiek aan die op regel :429 (die volgens mij ook binnen een describe zou moeten vallen). Moest in de review goed opletten waarom deze 2 tests niet identiek aan elkaar waren.

Reply:
Deze zit al binnen de `describe("subscribeDevice()")`
![image](https://user-images.githubusercontent.com/26112463/176125962-f64d0769-30fb-4818-8829-c5ccf181cb37.png)

regel :429 test hetzelfde maar die via een andere route (`sendMessage()` functie)
![image](https://user-images.githubusercontent.com/26112463/176126165-26349f31-d52f-4690-a824-0b790586e1f3.png)

Dat lijkt me genoeg van elkaar te verschillen toch? Als we bij 1 van de 2 functies vergeten om `referer` mee te geven dan zien we dat hier terugkomen.

Reply:
Je hebt helemaal gelijk maar het werd vreemd gerenderd in github waardoor ik dacht dat die subscribe ervoor stopte

![image](https://user-images.githubusercontent.com/4190722/176297789-fee1ee8a-db4b-45ba-a2ca-0970067d00ac.png)